eng/docker/Readme.md

```diff
@@ -5,7 +5,7 @@ Provides reusable docker build infrastructure for the dotnet/runtime repo.
 ## libraries-sdk Dockerfiles
```

Comment:
GitHub doesn't render the diff until I explicitly click "Load diff" for some files, including this one. (It doesn't have a large changes, so that's not the reason)

Could it be some encoding problem?

cc: @sharwell, I think you referred in a comment somewhere that GitHub treats some files as binary if they have incorrect encoding. Can you confirm?

Reply:
github does this in PRs that modify many files. I think it is the case here.